src/steamship_langchain/vectorstores/steamship_vector_store.py

```diff
@@ -90,6 +153,8 @@ def add_texts(self, texts: Iterable[str], metadatas: Optional[List[dict]] = None
 
         self.index.insert(items)
 
+        return [str(uuid.uuid1()) for _ in texts]
```

Comment:
What's the purpose of these random IDs? We can't use them to retrieve the files right?

Reply:
They are part of the VectorStore contract at the moment (https://github.com/hwchase17/langchain/blob/master/langchain/vectorstores/base.py#L15-L20).

I've updated the code to set the ids to be ids of the Tags. That _should_ make them somewhat (?) relevant, but since we don't expose id-based removal from the store in our current abstraction, I'm not sure it matters in the end?